components/producers/ossf-scorecard/Makefile

```diff
@@ -2,7 +2,7 @@
 
 CONTAINER_REPO=
 DRACON_VERSION=
-OSSF_SCORECARD_SAFETY_BASE_IMAGE=$(shell test -e .custom_image && cat .custom_image || echo "alpine:latest")
+OSSF_SCORECARD_SAFETY_BASE_IMAGE="alpine:latest"
```

Comment:
I don't think we need this at all tbh. We can parameterise the image path in the task and if a user wants a different image with scorecard they can set that in their pipeline runs, same way we do it in the git-clone task